Source/Management.GraphQL/EventHandlers/EventHandler.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.Runtime.Management.GraphQL.EventHandlers
+{
+    public class EventHandler
+    {
+        public Guid Id { get; set; }
+        public Guid Scope { get; set; }
+        public int FilterPosition { get; set; }
+        public int EventProcessorPosition { get; set; }
+        public int TailEventLogSequenceNumber { get; set; }
+    }
```

Comment:
There is also information available in the event store for when the stream processors last processed an event - and if it is failing (and in that case why they failed (the exception message in most cases). That would be nice to see here?